src/components/actions/index.jsx

```diff
@@ -1,32 +1,41 @@
-import React from "react";
+import React, { useEffect } from "react";
 import PropTypes from "prop-types";
 import { Link } from "gatsby";
 
 import { LAYOUTS, SECTIONS } from "src/constants";
 
 import "./index.scss";
 
-const Actions = ({ actions, activeAction }) => (
-  <ul className="actions">
-    {actions.map((action, index) => (
-      <li
-        key={`${action.route}-${index}`}
-        className={`actions__item ${
-          activeAction === action ? "actions__item--active" : ""
-        }`}
-      >
-        <Link
-          data-section={SECTIONS.ACTIONS}
-          data-layout={LAYOUTS.LIST}
-          to={action.route}
-          aria-label={`${action.label} vim color schemes`}
+const Actions = ({ actions, activeAction }) => {
+
+  useEffect(() => {
+    document.querySelector('.actions__item--active')
+      .scrollIntoView(false);
+  }, []);
```

Comment:
Nice! I wouldn't have thought of that.

I'm surprised that Gatsby is able to build that. Whenever I want to work with `document` or `window` within Gatsby, it screams at me because, at build time, they are not defined. It forces me to do checks like:
```js
if (typeof document !== "undefined) {
  // do stuff with document
}
```

It seems to be working for now, though. Vercel was able to deploy.

Reply:
I think it's because the `useEffect` hook is not executed on the server side. At least from my experience with Next.js.

Reply:
Makes sense!